tabs/src/services/lnbitsServiceLocal.ts

```diff
@@ -500,6 +500,32 @@ const getWalletIdByUserId = async (adminKey: string, userId: string) => {
   }
 };
 
+const getRewards = async (apiKey: string) => {
+  console.log('Getting products ...');
+  try {
+    const response = await fetch('https://demo.lnbits.com/market/api/v1/products?all_stalls=false', {
```

Comment:
Check how the other methods work here (environment variable for the Url) with an entry in setupProxy.js for routing to stop CORS errors. (Ref: https://demo.lnbits.com/market/api/v1/products)

Reply:
Looks like you still have the LNBits instant hardcoded. See other functions don't use this.

Reply:
getRewards is not actual function. I've removed it and replaced with getNostrRewards

![image](https://github.com/user-attachments/assets/cbb01035-0262-4307-99b2-4a9a5d48ea5d)

Reply:
And using Promises / Types xxx